.github/workflows/packaging.yml

```diff
@@ -73,13 +73,13 @@ jobs:
           # uses the libcurl implementation provided by tarantool
           # executable and when the module falls back to the
           # system libcurl.
-          #
-          # TODO: Enable 1.10.7 jobs after resolving gh-44
-          # ('Unable to load libcurl.so'). Don't forget to exclude
-          # distribution versions, which are not supported by
-          # tarantool 1.10.7.
-          # - release/1.10.7
+          - release/1.10.7
           - live/2.9
+        exclude:
+          - platform: {os: debian, dist: bullseye}
+            tarantool: release/1.10.7
+          - platform: {os: fedora, dist: 34}
+            tarantool: release/1.10.7
```

Comment:
Please, leave a comment with the reason of excluding. It may be not so obvious.

Reply:
Added.